IntegrationTests/Outer API Tests For Standards.postman_environment.json

```diff
@@ -0,0 +1,34 @@
+{
+	"id": "12cbe017-af61-4ce5-b734-2f6a057ccfad",
+	"name": "Test",
+	"values": [
+		{
+			"key": "test-environment",
+			"value": "test2",
+			"enabled": true
+		},
+		{
+			"key": "courses-api-client-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "courses-api-client-secret",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "courses-api-resource",
+			"value": "https://citizenazuresfabisgov.onmicrosoft.com/das-test2-crsapi-as-ar",
```

Comment:
this kind of information should not be in this repo

Reply:
Removed